docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.3'
+
+services:
+   db:
+     image: mysql:5.7
+     volumes:
+       - db_data:/var/lib/mysql
+     restart: always
+     environment:
+       MYSQL_ROOT_PASSWORD: m4ster_s3cret
+       MYSQL_DATABASE: libreehr
+       MYSQL_USER: libreehr
+       MYSQL_PASSWORD: s3cret
+
+   librehealth:
+     depends_on:
+       - db
```

Comment:
Add a healthcheck. See comments on old outdated diff for resources.

Reply:
I would say that is not needed in this case. `depends_on` being sufficient.

Will add it if you prefer, though.

Reply:
a healthcheck is a good thing to have anyways -- lets us know when the db is not healthy